plugin/trino-hive/src/test/java/io/trino/plugin/hive/BaseHiveConnectorTest.java

```diff
@@ -2175,12 +2175,12 @@ public void testFilterOnBucketedTable(BucketedFilterTestSetup testSetup)
         String tableName = "test_filter_on_bucketed_table_" + randomTableSuffix();
         assertUpdate(
                 """
-                CREATE TABLE %s (bucket_key %s, other_data double)
-                WITH (
-                    format = 'TEXTFILE',
-                    bucketed_by = ARRAY[ 'bucket_key' ],
-                    bucket_count = 5)
-                """.formatted(tableName, testSetup.getTypeName()));
```

Comment:
These changes looks wrong. Please drop this commit.

Reply:
This is autoformatting by IntelliJ...

Reply:
It looks wrong. We probably need to update code style for text blocks. Let's revert this for now, since the original formatting looks like the desired formatting.